pkg/webservice/handlers.go

```diff
@@ -378,6 +385,20 @@ func updateConfig(w http.ResponseWriter, r *http.Request) {
 	buildUpdateResponse(true, "", w)
 }
 
+func getConfigurationString(requestBytes []byte, checksum [32]byte) string {
+	newConf := string(requestBytes)
+	checksumString := fmt.Sprintf("%v", checksum)
+	checksumString = strings.ReplaceAll(checksumString, " ", ",")
+	checksumString = "checksum: " + checksumString
```

Comment:
In the `getClusterConfig()` we use the following string to write the checksum:
`sha256 := fmt.Sprintf("\nsha256 checksum: %X", conf.Checksum)`
The `%X` converts the checksum into a hex string which is the same for both json and yaml. 

I cannot assess if this works for both or how this relates to the current layout we have in the get call.

Reply:
That will not work for this case, because in this case we need to have an array with the checksum bytes delimited by ",". In getClusterConfig() it will result in a human readable checksum string